vscode/src/telemetry.ts

```diff
@@ -250,3 +250,19 @@ export function sendTelemetryEvent<E extends keyof EventTypes>(
     )}`,
   );
 }
+
+function getBrowserRelease(): string {
+  if (navigator.userAgentData) {
+    const browser =
+      navigator.userAgentData.brands[navigator.userAgentData.brands.length - 1];
```

Comment:
We discussed already in a separate thread that this isn't cross-browser available, and that we run in the browser. If I try and evaluate this expression in Safari I get the below. Please include the appropriate error handling so this doesn't throw exceptions.

<img width="601" alt="image" src="https://github.com/microsoft/qsharp/assets/993909/4c295ad9-9e09-4ed4-8490-9967d42c45f8">

Reply:
Doh! How'd I miss that. Yeah, should do it.

Just in case some browser (ahem... Safari) ever adds 'userAgentData' but still doesn't add the child properties like 'brands', might want to change the condition on 255 to be `navigator.userAgentData?.brands` for extra safety).